db/seeds/object_manager_attributes.rb

```diff
@@ -196,7 +196,6 @@
         filter: Ticket::State.by_category(:viewable_agent_new).pluck(:id),
       },
       'ticket.customer' => {
-        item_class: 'column',
```

Comment:
Why was this line removed?

Reply:
it's needed to fit 2 select (Ticket.state and Ticket.organization_id) in 1 row